frontend/src/components/common/info/messages/EmptyDistributeClassroomsMessage.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { Center } from "@chakra-ui/react";
+
+import { PlusOutlineIcon } from "../../../../assets/icons";
+import DisplayAssessmentsIllustration from "../../../../assets/illustrations/display-assessments.svg";
+import MessageContainer from "../MessageContainer";
+
+type EmptyDistributeClassroomsStateProps = {
+  onClick?: () => void;
+};
+
+const EmptyDistributeClassroomsMessage = ({
+  onClick,
+}: EmptyDistributeClassroomsStateProps): React.ReactElement => {
+  return (
+    <Center
```

Comment:
we have a lot of repeats of this in the codebase (like 5) - could we make a theme variant for `<Center>`?

Reply:
good ideal! will do

Reply:
implemented this according to: https://github.com/chakra-ui/chakra-ui/issues/2456

let me know if there's a more straightforward way of doing this 🤔

Reply:
ah shame.. should be ok then. ig one other way is to make a custom <Center> component